tests/www/apply/test_submit.py

```diff
@@ -4361,44 +4408,21 @@ def test_with_previous_as_job_seeker(self, client):
         response = client.post(self.check_prev_applications_url, data={"force_new_application": "force"})
         assertRedirects(response, self.application_jobs_url)
 
-        # Check that the back URL is correct
+        # Check that the reset URL is correct
```

Comment:
nit: Check that... n’est pas informatif.
```suggestion
        # Reset URL is correct
```